vexim/adminaliaschangesubmit.php

```diff
@@ -43,7 +43,7 @@
   }
 
   # Update the password, if the password was given
-  if(isset($_POST['password']) && $_POST['password']!='' ){
+  if (isset($_POST['password'])) {
```

Comment:
According to the comment above, here `$_POST['password']` could be set to an empty value, which indicates that it shouldn't be changed.
I think we should check for existence and non-emptiness of both `$_POST['password']` and `$_POST['vpassword']` here, and only proceed if all four conditions are satisfied.